test/e2e/networks/dashcore.toml

```diff
@@ -69,42 +73,25 @@ perturb = ["pause"]
 
 [node.validator05]
 start_at = 1005 # Becomes part of the validator set at 1010
-seeds = ["seed02"]
+seeds = ["seed01"]
 database = "cleveldb"
-fast_sync = "v0"
-# FIXME: should be grpc, disabled due to https://github.com/tendermint/tendermint/issues/5439
-#abci_protocol = "grpc"
+block_sync = "v0"
+state_sync = "p2p"
 privval_protocol = "dashcore"
 perturb = ["kill", "pause", "disconnect", "restart"]
 
 [node.full01]
 start_at = 1010
 mode = "full"
-# FIXME: should be v2, disabled due to flake
-fast_sync = "v0"
-persistent_peers = ["validator01", "validator02", "validator03", "validator04", "validator05"]
-privval_protocol = "dashcore"
-retain_blocks = 1
-perturb = ["restart"]
-
-[node.full02]
-start_at = 1015
-mode = "full"
-# FIXME: should be v2, disabled due to flake
-fast_sync = "v0"
-state_sync = true
+block_sync = "v0"
+state_sync = "rpc"
+persistent_peers = ["validator01", "validator02", "validator03", "validator04"]
 privval_protocol = "dashcore"
-seeds = ["seed01"]
+retain_blocks = 10
 perturb = ["restart"]
 
 [node.light01]
 mode= "light"
-start_at= 1005
+start_at= 1010
 privval_protocol = "dashcore"
 persistent_peers = ["validator01", "validator02", "validator03"]
-
-[node.light02]
```

Comment:
should we restore light nodes?

Reply:
the idea is to have the same configuration with upstream